package.json

```diff
@@ -28,8 +31,11 @@
     "uuid": "^8.3.2"
   },
   "devDependencies": {
+    "@shelf/jest-mongodb": "^2.0.3",
```

Comment:
Do you need this dependency? It seems unused since we populate the test database and everything ourselves

Reply:
removed it, I had been investigating using that to populate the database